docs/core/docker/publish-as-container.md

```diff
@@ -182,6 +182,46 @@ You can control many aspects of the generated container through MSBuild properti
 > [!NOTE]
 > The only exceptions to this are `RUN` commands. Due to the way containers are built, those cannot be emulated. If you need this functionality, you'll need to use a _Dockerfile_ to build your container images.
 
+### Container image naming configuration
+
+Container images follow a specific naming convention. The name of the image is composed of several parts, the registry, optional port, repository, and optional tag and family.
+
+```dockerfile
+REGISTRY[:PORT]/REPOSITORY[:TAG(S)[-FAMILY]]
+```
+
+For example, `mcr.microsoft.com/dotnet/runtime:8.0-alpine`.
+
+Some properties described in the following sections correspond to managing parts of the image name. Consider the following table that maps the relationship between the image name and the properties:
+
+:::zone pivot="dotnet-8-0"
+
+| Image name part   | MSBuild property      | Example                                       |
+|-------------------|-----------------------|-----------------------------------------------|
+| `REGISTRY[:PORT]` | `ContainerRegistry`   | `mcr.microsoft.com:443`                       |
+| `PORT`            | `ContainerPort`       | `:443`                                        |
+| `REPOSITORY`      | `ContainerRepository` | `dotnet/runtime`                              |
+| `REPOSITORY`      | `ContainerImageName` | `dotnet/runtime`                              |
```

Comment:
How bad does it look if we do something like this?

```suggestion
| `REPOSITORY`      | `ContainerRepository` or `ContainerImageName` (deprecated) | `dotnet/runtime`                              |
```

Reply:
This is wrapped in a zone pivot, .NET 7 gets the `ContainerImageName` and .NET 8 gets the `ContainerRepository`. I think that's the best way forward... agree?

Reply:
I've fixed this, it was a copy paste error since I intended to have them in zones.